bluesky/examples.py

```diff
@@ -154,8 +153,14 @@ def __getstate__(self):
         return (self.name, self._fields, self.read_attrs, self.conf_attrs,
                 self._monitor_intervals)
 
-    def trigger(self):
-        "No-op: returns a status object immediately marked 'done'."
+    def trigger(self, *, delay_time=0):
```

Comment:
to avoid picking a magic attribute name and spending too much time walking the class tree.